linkml/generators/pydanticgen/pydanticgen.py

```diff
@@ -187,10 +187,7 @@ class PydanticGenerator(OOCodeGenerator):
             Path,
             PurePath as RenamedPurePath
         )
-        if sys.version_info >= (3, 8):
-            from typing import Literal
-        else:
-            from typing_extensions import Literal
```

Comment:
oh my bad for missing this before! This is a comment demonstrating the usage and results of this object, not the template, so it's correct! 

I pre-emptively removed typing_extensions from the [`DEFAULT_IMPORTS`](https://github.com/linkml/linkml/blob/581bc8091bc87630d72a0930de12b8bad4d153d3/linkml/generators/pydanticgen/pydanticgen.py#L63) assuming this would be merged shortly after :) i'll fix rn